src/devtools/client/debugger/src/components/App.css

```diff
@@ -1,8 +1,8 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
-button {
+.button {
```

Comment:
why do we need to do this?

Reply:
we don't. it was from my CSS fix exploration. i'll revert